canonical-web-and-design/maas-ui
# feat(ui): display installation status

## Done

- Display the installation status in the installation log.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Find a commissioned machine and click "Deploy" in the action menu.
- Visit the React Logs tab (visit machine details and change the URL from '/summary' to '/logs') and click "Installation output".
- You should see different statuses depending on what the machine is doing.

## Fixes

Fixes: #canonical-web-and-design/maas-squad#2375.
